minigotchi/minigotchi.ino

```diff
@@ -21,23 +21,29 @@ void setup() {
  */
 
 void loop() {
+
+    Parasite::readData();
```

Comment:
We can probably move Parasite::readData() to the functions that handle the tasks in the loop, which are in `minigotchi.cpp`. I intended for them to be modified just in case for a time like this.

Reply:
Done, moved them to the main methods
Also added Parasite::sendName() to minigotchi's boot() method in case someone hooks up the minigotchi after the pwnagotchi has been powered on